### ISSUE TITLE
[BUG] cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory when running security tests that rely on kubescape

### ISSUE THREAD

**Issue by agentpoyo** [cncf/cnf-testsuite#1659]
**Describe the bug**
When running cert, security tests are failing with kubescape json results file not found errors on RHEL v8

```
cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
Error resolving real path: 'cnf-testsuite': No such file or directory
```

**To Reproduce**
Setup cnf-testsuite on RHEL 8 and run the `cert` command.

You should see errors for security tests not finding the kubescape json results file:

```
Security Tests
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:269:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:266:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:247:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:244:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:383:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:380:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:407:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:404:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:359:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:356:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:431:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:428:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:455:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:452:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:479:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:476:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:503:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:500:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:528:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:525:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
/usr/share/crystal/src/crystal/system/unix/file.cr:97:5 in 'tools_path'
/workspace/src/tasks/utils/kyverno.cr:11:16 in 'install'
/workspace/src/tasks/workload/security.cr:136:3 in '->'
/workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
/workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
/workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
/usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
Error resolving real path: 'cnf-testsuite': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: Error opening file with mode 'r': 'kubescape_results.json': No such file or directory
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/system/unix/file.cr:11:7 in 'open'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/file.cr:111:10 in 'parse'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:552:17 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:549:3 in '->'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-09-28 12:36:34 -07:00 #3915088] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
```

**Expected behavior**
Security tests that rely on kubescape should run without errors.

**Comment by HashNuke** [comment]
I tried to reproduce this issue and then found another issue with the security category on RHEL8. Created a ticket to fix #1666.

**Comment by denverwilliams** [comment]
@agentpoyo I was unable to reproduce. Can we close this now?
